episodes/contact-matrices.Rmd

```diff
@@ -0,0 +1,228 @@
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+The example above uses the POLYMOD survey. There are a number of surveys available in `socialmixr`, to list the available surveys use `list_surveys()`. To download a survey, we can use `get_survey()`
+
+```{r, message = FALSE, warning = FALSE}
+zambia_sa_survey <- get_survey("https://doi.org/10.5281/zenodo.3874675")
+```
+
+
+
+::::::::::::::::::::::::::::::::::::: challenge 
+
+## Zambia contact matrix
+
+After downloading the survey, generate a symmetric contact matrix for Zambia using the following age bins:
+
++ [0,15)
++ 20+
+
+:::::::::::::::::::::::: solution 
+
+```{r polymod_poland}
+contact_data_zambia <- contact_matrix(
+  survey = zambia_sa_survey,
+  age.limits = c(0, 20),
+  symmetric = TRUE
+)
+contact_data_zambia
+```
+:::::::::::::::::::::::::::::::::
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+
+## Analyses with contact matrices
+
+Contact matrices can be used in a wide range of epidemiological analyses, they can be used:
+
++ to calculate the basic reproduction number while accounting for different rates of contacts between age groups [(Funk et al. 2019)](https://doi.org/10.1186/s12916-019-1413-7),
++ to calculate final size of an epidemic, see the R package `{finalsize}`,
++ to assess the impact of interventions finding the relative change between pre and post intervention contact matrices to calculate the relative difference in $R_0$ [(Jarvis et al. 2020)](https://doi.org/10.1186/s12916-020-01597-8),
++ and in mathematical models to account for group specific contact patterns (see next section.)
+
+::::::::::::::::::::::::::::::::::::: callout
+## Normalisation
+
+There are times when we need to change the scale of our contact matrices e.g. to compare matrices to model projected matrices [(Prem et al. 2021)](https://doi.org/10.1371/journal.pcbi.1009098) the entries need to be on the same scale.
+
+Normalisation means converting a value to be between 0 and 1.To normalise a matrix we scale the matrix so that the largest eigenvalue is 1. This transformation scales the entries between 0 and 1 but preserves the relative differences between the entries. To perform this normalisation we divide all entries of the matrix by the largest eigenvalue of the matrix.
+
+```{r}
+contact_matrix <- t(contact_data$matrix)
+contact_matrix <- contact_matrix / max(eigen(contact_matrix)$values)
+contact_matrix
```

Comment:
```suggestion
normalised_matrix <- contact_matrix*scaling_factor

r0 <- 2
beta <- r0*scaling_factor*gamma
```
Not sure this is optimal in terms of R syntax vs equations, but hopefully on the right track in communication terms!